lib/studies/reply/adaptive_nav.dart

```diff
@@ -1135,11 +1147,16 @@ class _ReplyFabState extends State<_ReplyFab>
                 ),
               );
             } else {
-              return FloatingActionButton(
-                heroTag: 'Bottom App Bar FAB',
-                tooltip: tooltip,
-                child: fabSwitcher,
-                onPressed: openContainer,
+              return InkWell(
```

Comment:
Can you keep the tooltip behavior by wrapping this with a `ToolTip` widget

Reply:
Will do.